rollout_captures_asg.sh

```diff
@@ -0,0 +1,256 @@
+#!/bin/bash
+
+set -euo pipefail
+
+OVERPROVISIONER_NAMESPACE=cluster-overprovisioner
+OVERPROVISIONER_DEPLOYMENT=cluster-overprovisioner-captures-overprovisioner
+CAPTURE_NAMESPACE=reference
+CAPTURE_IMAGE=$(
+  kubectl -n $CAPTURE_NAMESPACE get deployments \
+    -l app.kubernetes.io/name=capture \
+    -o jsonpath='{.items[0].spec.template.spec.containers[0].image}'
+)
+
+# Colors
+end="\033[0m"
+black="\033[0;30m"
+blackb="\033[1;30m"
+white="\033[0;37m"
+whiteb="\033[1;37m"
+red="\033[0;31m"
+redb="\033[1;31m"
+green="\033[0;32m"
+greenb="\033[1;32m"
+yellow="\033[0;33m"
+yellowb="\033[1;33m"
+blue="\033[0;34m"
+blueb="\033[1;34m"
+purple="\033[0;35m"
+purpleb="\033[1;35m"
+lightblue="\033[0;36m"
+lightblueb="\033[1;36m"
+
+function green {
+  echo -e "${green}${1}${end}"
+}
+
+function greenb {
+  echo -e "${greenb}${1}${end}"
+}
+
+function white {
+  echo -e "${white}${1}${end}"
+}
+
+function whiteb {
+  echo -e "${whiteb}${1}${end}"
+}
+
+function yellow {
+  echo -e "${yellow}${1}${end}"
+}
+
+function yellowb {
+  echo -e "${yellowb}${1}${end}"
+}
+
+function redb {
+  echo -e "${redb}${1}${end}"
+}
+
+function fail_and_exit {
+  redb "$1"
+  exit 1
+}
+
+function drain {
+  yellow "Draining $1"
+  kubectl drain $1 --ignore-daemonsets=true --delete-emptydir-data=true --timeout=120s --skip-wait-for-delete-timeout=1
+  green "$1 successfully drained."
+}
+
+function releaseip {
+  local pending_pods_count
+  externalip_name=$(
+    kubectl get externalips \
+      -o jsonpath="{.items[?(@.spec.nodeName==\"$1\")].metadata.name}"
+  )
+
+  for ip in $externalip_name; do
+    green "Disassociate EIP $ip from $1"
+    kubectl patch externalips $ip --type=merge -p '{"spec": {"nodeName": ""}}'
+  done
+}
+
+function kubestatic_unlabel {
+  green "Remove kubestatic label on node $1"
+  kubectl label node $1 kubestatic.quortex.io/externalip-auto-assign-
```

Comment:
Could there be a mistake on the label?

Reply:
It is the syntax to remove a label : https://kubernetes.io/docs/reference/kubectl/cheatsheet/#updating-resources